mypy.ini

```diff
@@ -1,5 +1,35 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
+
+# - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
+# - All jaraco modules are still untyped
+# - wheel appears to be untyped
```

Comment:
If wheel is still untyped, this comment should point to a ticket tracking getting it typed.

Reply:
It is inherently wrong to say "untyped". I opened the code and there are a lot of type hints.
What is missing is the `py.typed` marker and some functions are truly untyped--and that's what needs some contribution.

Reply:
@bswck I copied the comment from setuptools. But yes "marked as `py.typed`" is the verbage I usually prefer.

Reply:
Am I right in understanding that importing `wheel` is a setuptool-specific concern ?
https://github.com/pypa/wheel/pull/620/files

Sounds like the package is becomming only a CLI tool and the functionality split across `packaging` and `setuptools`.